platform-sdk/settings.gradle.kts

```diff
@@ -90,6 +90,14 @@ dependencyResolutionManagement {
       version("lazysodium-version", "5.1.1")
       version("bouncycastle-version", "1.70")
 
+      version("hedera-bls-api-version", "0.1.0-SNAPSHOT")
```

Comment:
Do these dependencies have Linux and MacOS support for both x86_64 and ARM64 architectures?

Reply:
No, not yet